.devcontainer/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM mcr.microsoft.com/vscode/devcontainers/go:0-1.19
+FROM mcr.microsoft.com/vscode/devcontainers/go:1-1.19
```

```suggestion
FROM mcr.microsoft.com/vscode/devcontainers/go:1-1.21
```

@JoelSpeed can you commit this before merging?